app-service-template/res/configuration.toml

```diff
@@ -62,13 +62,11 @@ Timeout = "30s"
 Type = 'vault'
 Host = 'localhost'
 Port = 8200
-Path = '/v1/secret/edgex/appservice/'
+Path = 'appservice/'
```

Comment:
I noticed in your code that adds the base you also add the ending `/` so we can drop that from here?

Reply:
we could, the reason that i keep / at then so it looks more like a relative path.

Reply:
K, as long as it functions I am good... :-)

Reply:
Either makes sense but on first read I do think the trailing slash suggests that you can go 'deeper' if needed.

Reply:
actually you could if you like: so appservice/example/ is ok in theory